Error when building with Visual Studio version prior to 15.5

**Describe the bug**
Without Visual Studio 15.5+, the gulp build will fail with multiple MSBuild errors due to not supporting the `/restore` flag.
```
MSBUILD : error MSB1001: Unknown switch.
Switch: /restore
For switch syntax, type "MSBuild /help"
[11:00:46] MSBuild failed with code 1!
[11:00:46] 'Build-Solution' errored after 255 ms
[11:00:46] Error: MSBuild failed with code 1!
    at ChildProcess.<anonymous> (E:\Paragon\dev\ParagonSitecore9TrainingSite\HabitatTrainingSite\node_modules\gulp-msbuild\lib\msbuild-runner.js:66:25)
    at emitTwo (events.js:87:13)
    at ChildProcess.emit (events.js:172:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
```

**To Reproduce**
Attempt to build with an earlier version of Visual Studio.

**Expected behavior**
Ideally, a message indicating that Visual Studio needs to be upgraded. At the very least, documentation needs to be updated to state this prerequisite.

--- a/docs/02-Resources.md
+++ b/docs/02-Resources.md
@@ -5,7 +5,8 @@ These are the tools and practices currently used by the development team in Habi
 You must install these to get Habitat running on your computer:
 
 * [PowerShell version 5 or later](https://www.microsoft.com/en-us/download/details.aspx?id=50395)
-* [Visual Studio 2017](https://www.visualstudio.com/downloads/) and .NET 4.6.
+* [Visual Studio 2017](https://www.visualstudio.com/downloads/) and .NET 4.7.1.
+  * **Be sure you are using VS17 v15.5 or higher in order to get necessary updates to MSBuild. Update your IDE if needed.**
   * Within VS2017, install these extensions:
   * [Web Compiler](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.WebCompiler)
 * ASP.NET MVC for the user interface patterns

